util/fibers/detail/fiber_interface.cc

```diff
@@ -290,6 +291,20 @@ ctx::fiber_context FiberInterface::SwitchTo() {
   });
 }
 
+void FiberInterface::PrintStackTrace() {
+  entry_ = std::move(entry_).resume_with([name = name_](ctx::fiber_context&& c) {
+    std::string stacktrace = GetStacktrace(false);
+    LOG(INFO) << "Fiber " << name << ":\n" << stacktrace;
```

Comment:
I'd add as much information on the fiber as possible, maybe epoch? I realize we don't have IDs, but perhaps that's also worth adding..

Reply:
We do have names

Reply:
I'm not saying we should rename the name (which Roy's already printing :)